lua/nvim-lsp-installer/fs.lua

```diff
@@ -21,9 +21,11 @@ end
 function M.rmrf(path)
     log.debug("fs: rmrf", path)
     assert_ownership(path)
-    if vim.fn.delete(path, "rf") ~= 0 then
-        log.debug "fs: rmrf failed"
-        error(("rmrf: Could not remove directory %q."):format(path))
+    if vim.fn.exists(path) then
```

Comment:
Although this may be the sensible thing to do, it breaks the current contract of this function - where it errors if the provided path does not exist. I think it'd be best to not introduce this diff